modules/custom_operations/user_ie_extensions/tokenizer/python/openvino_tokenizers/__init__.py

```diff
@@ -16,38 +16,42 @@
 
 
 _extension_path = os.environ.get("OV_TOKENIZER_PREBUILD_EXTENSION_PATH")
-_ext_name = "user_ov_extensions"
 if _extension_path:
-    # when the path to extension set manually
+    # when the path to the extension set manually
     _ext_libs_path = Path(_extension_path).parent
 else:
     # python installation case
     _ext_libs_path = Path(sysconfig.get_paths()["purelib"]) / __name__ / "libs"
 
+_ext_name = "user_ov_extensions"
 if sys.platform == "win32":
-    _ext_path = _ext_libs_path / f"{_ext_name}.dll"
-    if _ext_libs_path.is_dir():
-        # On Windows, with Python >= 3.8, DLLs are no longer imported from the PATH.
-        os.add_dll_directory(str(_ext_libs_path.absolute()))
-    else:
-        sys.exit(f"Error: extention libriary path {_ext_libs_path} not found")
+    _ext_name = f"{_ext_name}.dll"
 elif sys.platform == "darwin":
-    _ext_path = _ext_libs_path / f"lib{_ext_name}.dylib"
+    _ext_name = f"lib{_ext_name}.dylib"
 elif sys.platform == "linux":
-    _ext_path = _ext_libs_path / f"lib{_ext_name}.so"
+    _ext_name = f"lib{_ext_name}.so"
+else:
+    sys.exit(f"Error: extension does not support the platform {sys.platform}")
+
+# conda-forge case
+conda_forge_lib_path = Path(os.path.join(os.path.dirname(__file__), "..", "..", ".."))
```

Comment:
Maybe it would be better to search extension in the os.environ["PATH"]?

Reply:
if conda env is not activated, it will not work.

Reply:
potentially I can use /abs/path/to/conda/env/python and openvino_tokenizers will find libraries.

Reply:
It should be more robust. We had problems with `pip install -e .` when trying to find libraries relative to the `__file__` path. There may be similar problems with a conda installation.

Reply:
conda does not work in editable mode